scans/auth/plans_and_scripts/testfire/csa.yaml

```diff
@@ -17,8 +17,8 @@ env:
     users:
     - name: testuser
       credentials:
-        Username: ${testfire_user}
-        Password: ${testfire_pass}
+        username: ${zapusername}
+        password: ${zappassword}
```

Comment:
The case matters (if the script were actually using the parameters).

Reply:
Tweaked

Reply:
Only applies to scripts.

Reply:
Done